quickstart/pom.xml

```diff
@@ -145,6 +148,20 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <source>1.8</source>
+              <target>1.8</target>
+              <annotationProcessorPaths>
+                <path>
+                  <groupId>com.datastax.oss.quarkus</groupId>
```

Comment:
This should actually reference the `cassandra-quarkus-mapper-processor` artifact. I also fear that we might end up with two implementations of `AnnotationProcessor`: 

1. `com.datastax.oss.quarkus.runtime.internal.mapper.processor.QuarkusMapperProcessor`
2. `com.datastax.oss.driver.internal.mapper.processor.MapperProcessor`

And both of them will execute. I think that for this reason, it's safer to specify which one we want:

```
              <annotationProcessors>
                <annotationProcessor>com.datastax.oss.quarkus.runtime.internal.mapper.processor.QuarkusMapperProcessor</annotationProcessor>
              </annotationProcessors>
```

Reply:
I'm again having trouble with javapoet, is it really required to exclude it? I re-included it back again and everything works... until we get here :-D.